jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/DockerContextMojoTest.java

```diff
@@ -259,4 +273,10 @@ private String getBaseImage() throws IOException {
     List<String> lines = Files.readAllLines(dockerfile);
     return lines.stream().filter(line -> line.startsWith("FROM")).findFirst().get();
   }
+
+  private String getCmd() throws IOException {
+    Path dockerfile = projectRoot.getRoot().toPath().resolve("target/Dockerfile");
+    List<String> lines = Files.readAllLines(dockerfile);
+    return lines.stream().filter(line -> line.startsWith("CMD")).findFirst().get();
+  }
```

Comment:
How about removing code duplication?

```java
private String getEntrypoint() {
  getDockerfileLine("ENTRYPOINT");
}

private String getBaseImage() {
  getDockerfileLine("FROM");
}

private String getCmd() {
  getDockerfileLine("CMD");
}

private String getDockerfileLine(String command) {
  Path dockerfile = projectRoot.getRoot().toPath().resolve("target/Dockerfile");
  List<String> lines = Files.readAllLines(dockerfile);
  return lines.stream().filter(line -> line.startsWith(command)).findFirst().get();
}
```

Reply:
done

Reply:
Let's do this for `DockerContextTaskTest` as well.

Reply:
yes obviously.